folly/EventPortability.cpp

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
From @yfeldblum any reason not to put the portability layer under `folly/io/async` instead of up-leveled here?